samples/CONFIG-MIGRATION.md

```diff
@@ -14,7 +15,11 @@ istioctl create route-rule -f myrule.yaml
 ```
 0.2.x:
 ```
-kubectl apply -f myrule.yaml
+istioctl create -f myrule.yaml
+
+    or (for Kubernetes users):
```

Comment:
This is just confusing, kubectl will work for anything, and it is secure, by comparison with istioctl which is not.